src/main/groovy/org/labkey/gradle/plugin/extension/DistributionExtension.groovy

```diff
@@ -22,12 +22,12 @@ import java.nio.file.Paths
 
 class DistributionExtension
 {
-    // the directory in which the file 'distribution' is placed, which contains the name of the distribution
-    // (used for mothership reporting and troubleshooting)
+    // the directory that holds the 'distribution.properties' file, which contains the name of the distribution (used
+    // by the webapp for mothership reporting, troubleshooting, and determining whether marketing information is shown),
+    // the distribution filename, and extraProperties from build.xml such as "supportedDatabases"
```

Comment:
I don't know that all this detail about the contents of the `distribution.properties` file is needed here. If needed, perhaps it belongs with the method that produces the file.

Reply:
Was just following the existing pattern, but okay... now slimmed down